src/tools/perf/api/libperf.h

```diff
@@ -116,30 +116,41 @@ typedef struct ucx_perf_result {
 } ucx_perf_result_t;
 
 
+typedef unsigned (*ucx_perf_rte_group_size_func_t)(void *rte_group);
+typedef unsigned (*ucx_perf_rte_group_index_func_t)(void *rte_group);
+typedef void (*ucx_perf_rte_barrier_func_t)(void *rte_group,
+                                            void (*progress)(void *arg),
```

Comment:
should we also have typedef for ucx_perf_rte_progress_cb_t ?

Reply:
yep, why not